in.p1x.TanksOfFreedom.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id": "in.p1x.TanksOfFreedom",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "tanksoffreedom",
+  "finish-args": [
+     "--socket=x11",
+     "--socket=pulseaudio",
+     "--persist=.godot",
+     "--share=ipc",
+     "--share=network",
+     "--device=all"
+  ],
+  "build-options" : {
+    "no-debuginfo" : true
+  },
+  "modules": [
+    {
+      "name": "scons",
+      "buildsystem": "simple",
+      "cleanup": [ "*" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/scons/scons/2.5.1/scons-2.5.1.tar.gz",
+          "sha256": "0b25218ae7b46a967db42f2a53721645b3d42874a65f9552ad16ce26d30f51f2"
+        }
+      ],
+      "build-commands": [ "python setup.py install --prefix=/app" ]
+    },
+    "shared-modules/glu/glu-9.0.0.json",
+    {
+      "name": "godot",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/godotengine/godot/archive/2.1.4-stable.tar.gz",
+          "sha256": "07cf3b01367d5ea53805f144bc60711bd79efb53f1f88d57d6a706e6944de8d7"
+        }
+      ],
+      "build-commands": [
+        "scons platform=x11 \\
+          CCFLAGS=-I/app/include \\
+          prefix=/app \\
+          unix_global_settings_path=/app \\
+          verbose=yes \\
+          -j `nproc` \\
+          target=release_debug \\
+          use_llvm=yes \\
+          builtin_freetype=no \\
+          builtin_libmpcdec=yes \\
+          builtin_libogg=no \\
+          builtin_libpng=no \\
+          builtin_libtheora=no \\
+          builtin_libvorbis=no \\
+          builtin_libwebp=no \\
+          builtin_libvpx=no \\
+          builtin_zlib=no \\
+          builtin_opus=yes \\
+          builtin_speex=yes \\
+          builtin_squish=yes \\
+          pulseaudio=yes \\
+          udev=no \\
+          tools=no",
+          "strip bin/godot*",
```

Comment:
You shouldn't strip resulting binaries manually. Debug symbols are extracted to `Debug` extension.